### PR TITLE
Update VM Scale Set to VS 2019 Preview 3

### PR DESCRIPTION
**Pull request by CaseyCarter**
... and hopefully get the CI infrastructure up off the floor in the process.

### CODE REVIEW

**Comment by StephanTLavavej** [comment]
We will need to fix #773 (otherwise the CI will break whenever Python attempts to update a component and needs admin permission), and there are apparently other problems accumulating (was the pool deleted?).

**Comment by cbezault** [comment]
We should fix #773 but I don't think it should be causing failures.

File: azure-devops/provision-image.ps1 [hunk 1]
```diff
@@ -371,8 +371,14 @@ InstallLLVM $LlvmUrl
 InstallPython $PythonUrl
 InstallVisualStudio -Workloads $Workloads -BootstrapperUrl $VisualStudioBootstrapperUrl
 InstallCuda -Url $CudaUrl -Features $CudaFeatures
+
 Write-Host 'Updating PATH...'
 $environmentKey = Get-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment' -Name Path
+$Env:PATH="$($environmentKey.Path);C:\Program Files\CMake\bin;C:\Program Files\LLVM\bin"
```
Comment (cbezault) [inline azure-devops/provision-image.ps1:377]: What is this doing?

Comment (CaseyCarter) [reply azure-devops/provision-image.ps1:377]: Updating the PATH from the registry value so the python we just installed - which presumably added itself to the PATH - will be found.

**Comment by cbezault** [comment]
I don't know if you care but it might be worth taking the PowerShell functions from #1019

**Comment by CaseyCarter** [comment]
> I don't know if you care but it might be worth taking the PowerShell function from #1019

Sure - I'll steal it. (PowerShell is a monkey-see-monkey-do language for me, so I wanted to verify that I had a functional fix before investing effort into doing it correctly.)

**Pull request merged**